pycqed/analysis/measurement_analysis.py

```diff
@@ -1190,9 +1190,19 @@ def fit_data(self, print_fit_results=False, **kw):
         self.fit_res = ['', '']
         # It would be best to do 1 fit to both datasets but since it is
         # easier to do just one fit we stick to that.
+        # We make an initial guess of the Rabi period using both quadratures
+        data = np.sqrt(self.measured_values[0]**2+self.measured_values[1]**2)
```

Comment:
I guess the underlying assumption here is that standard IQ demodulation is used such that this will not amplify noise in one channel more than in the other. I think that assumption is fine at this time.